### PR TITLE
Add release notes for 11/7 2.14.4

### PR DESCRIPTION
**Pull request by zhen0**
Release notes for 2.14.4

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+- Add `SIGTERM` handling to runner to gracefully handle timeouts — https://github.com/PrefectHQ/prefect/pull/11133
+- Flow run graph v2 polish — https://github.com/PrefectHQ/prefect/pull/11105
+- Handle Cached tasks on graph-v2 — https://github.com/PrefectHQ/prefect/pull/11113
+- Exclude any flow/task run with a `NULL` `start_time` from Graph v2 — https://github.com/PrefectHQ/prefect/pull/11132
+- Allow tasks to use `get_run_logger` w/o parent flow run — https://github.com/PrefectHQ/prefect/pull/11129
+- Allow `ResultFactory` creation `from_task` when no `flow_run_context` — https://github.com/PrefectHQ/prefect/pull/11134
+- Use an unstarted flow run's expected start time for the Graph start time — https://github.com/PrefectHQ/prefect/pull/11138
+
+### Fixes
+- Dont print worker start command for managed pool — https://github.com/PrefectHQ/prefect/pull/11122
```
Comment (serinamarie) [inline RELEASE-NOTES.md:33]:
```suggestion
- Avoid printing references to workers when deploying to managed pools — https://github.com/PrefectHQ/prefect/pull/11122
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+
+### Documentation
+- Fix typo in RELEASE-NOTES.md — https://github.com/PrefectHQ/prefect/pull/11094
```
Comment (serinamarie) [inline RELEASE-NOTES.md:36]:
```suggestion
- Fix typo in release notes — https://github.com/PrefectHQ/prefect/pull/11094
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,46 @@
...
+- Fix docstring for `flow.deploy` method example — https://github.com/PrefectHQ/prefect/pull/11108
+- Add warning to push pool guide — https://github.com/PrefectHQ/prefect/pull/11118
```
Comment (serinamarie) [inline RELEASE-NOTES.md:38]:
```suggestion
- Add warning about image architecture to push pool guide — https://github.com/PrefectHQ/prefect/pull/11118
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,46 @@
...
+- Move `Webhooks` to `Development` — https://github.com/PrefectHQ/prefect/pull/11141
```
Comment (serinamarie) [inline RELEASE-NOTES.md:39]:
```suggestion
- Move webhooks guide to `Development` section in guides index — https://github.com/PrefectHQ/prefect/pull/11141
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,46 @@
...
+
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:41]:
```suggestion

```
one less blank line

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,46 @@
...
+- Allow `image` passed into `deploy` to be optional if loading flow from storage — https://github.com/PrefectHQ/prefect/pull/11117
+- Ensure client avoids image builds for managed work pools — https://github.com/PrefectHQ/prefect/pull/11120
+- Improved flow run graph with new dependency layout — https://github.com/PrefectHQ/prefect/pull/11112
+- Add `SIGTERM` handling to runner to gracefully handle timeouts — https://github.com/PrefectHQ/prefect/pull/11133
+- Flow run graph v2 polish — https://github.com/PrefectHQ/prefect/pull/11105
+- Handle Cached tasks on graph-v2 — https://github.com/PrefectHQ/prefect/pull/11113
+- Exclude any flow/task run with a `NULL` `start_time` from Graph v2 — https://github.com/PrefectHQ/prefect/pull/11132
+- Allow tasks to use `get_run_logger` w/o parent flow run — https://github.com/PrefectHQ/prefect/pull/11129
+- Allow `ResultFactory` creation `from_task` when no `flow_run_context` — https://github.com/PrefectHQ/prefect/pull/11134
+- Use an unstarted flow run's expected start time for the Graph start time — https://github.com/PrefectHQ/prefect/pull/11138
```
Comment (serinamarie) [inline RELEASE-NOTES.md:30]: maybe "in the flow run graph in the Prefect UI"

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,46 @@
...
+- Add a count blocks api route — https://github.com/PrefectHQ/prefect/pull/11090
+- Improved tag handling on DeploymentImage — https://github.com/PrefectHQ/prefect/pull/11115
+- Improve tag handling in `DeploymentImage` with `parse_image_tag` — https://github.com/PrefectHQ/prefect/pull/11119
+- Allow `image` passed into `deploy` to be optional if loading flow from storage — https://github.com/PrefectHQ/prefect/pull/11117
+- Ensure client avoids image builds for managed work pools — https://github.com/PrefectHQ/prefect/pull/11120
+- Improved flow run graph with new dependency layout — https://github.com/PrefectHQ/prefect/pull/11112
+- Add `SIGTERM` handling to runner to gracefully handle timeouts — https://github.com/PrefectHQ/prefect/pull/11133
+- Flow run graph v2 polish — https://github.com/PrefectHQ/prefect/pull/11105
+- Handle Cached tasks on graph-v2 — https://github.com/PrefectHQ/prefect/pull/11113
+- Exclude any flow/task run with a `NULL` `start_time` from Graph v2 — https://github.com/PrefectHQ/prefect/pull/11132
```
Comment (serinamarie) [inline RELEASE-NOTES.md:27]: These can be referenced in the highlight instead

File: RELEASE-NOTES.md (hunk 6 above)
Comment (serinamarie) [inline RELEASE-NOTES.md:30]: This one can be added to the highlight section as well

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,46 @@
...
+- Optimized api endpoint for fetching graph data
+- Horizontal scale controls
+
+### Enhancements
+- Add a count blocks api route — https://github.com/PrefectHQ/prefect/pull/11090
+- Improved tag handling on DeploymentImage — https://github.com/PrefectHQ/prefect/pull/11115
+- Improve tag handling in `DeploymentImage` with `parse_image_tag` — https://github.com/PrefectHQ/prefect/pull/11119
+- Allow `image` passed into `deploy` to be optional if loading flow from storage — https://github.com/PrefectHQ/prefect/pull/11117
+- Ensure client avoids image builds for managed work pools — https://github.com/PrefectHQ/prefect/pull/11120
+- Improved flow run graph with new dependency layout — https://github.com/PrefectHQ/prefect/pull/11112
```
Comment (serinamarie) [inline RELEASE-NOTES.md:23]: Same with this one

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,46 @@
...
+- Comparative duration layout
+- Cached tasks appear on the graph
+- Faster layouts
+- Optimized api endpoint for fetching graph data
+- Horizontal scale controls
+
+### Enhancements
+- Add a count blocks api route — https://github.com/PrefectHQ/prefect/pull/11090
+- Improved tag handling on DeploymentImage — https://github.com/PrefectHQ/prefect/pull/11115
+- Improve tag handling in `DeploymentImage` with `parse_image_tag` — https://github.com/PrefectHQ/prefect/pull/11119
```
Comment (serinamarie) [inline RELEASE-NOTES.md:20]: I think these can be combined like:

```suggestion
- Improved tag handling on `DeploymentImage` for `.deploy`
    - https://github.com/PrefectHQ/prefect/pull/11115
    - https://github.com/PrefectHQ/prefect/pull/11119
```

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,44 @@
...
+#### Features
+- Grid based dependency layouts
+- Comparative duration layout
+- Cached tasks appear on the graph
+- Faster layouts
+- Optimized api endpoint for fetching graph data
+- Horizontal scale controls
+
+### Enhancements
+- Add a count blocks api route — https://github.com/PrefectHQ/prefect/pull/11090
```
Comment (serinamarie) [inline RELEASE-NOTES.md:18]:
```suggestion
- Add API route for block counts — https://github.com/PrefectHQ/prefect/pull/11090
```

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,44 @@
...
+- Improved tag handling on DeploymentImage — https://github.com/PrefectHQ/prefect/pull/11115
+- Improve tag handling in `DeploymentImage` with `parse_image_tag` — https://github.com/PrefectHQ/prefect/pull/11119
+- Allow `image` passed into `deploy` to be optional if loading flow from storage — https://github.com/PrefectHQ/prefect/pull/11117
```
Comment (serinamarie) [inline RELEASE-NOTES.md:21]:
```suggestion
- Allow `image` passed into `.deploy` to be optional if loading flow from storage — https://github.com/PrefectHQ/prefect/pull/11117
```

Comment (serinamarie) [reply RELEASE-NOTES.md:21]: this helps differentiate from `prefect deploy` cli experience

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -1,5 +1,44 @@
...
+- Ensure client avoids image builds for managed work pools — https://github.com/PrefectHQ/prefect/pull/11120
```
Comment (serinamarie) [inline RELEASE-NOTES.md:22]:
```suggestion
- Ensure client avoids image builds when deploying to managed work pools — https://github.com/PrefectHQ/prefect/pull/11120
```

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -1,5 +1,43 @@
 # Prefect Release Notes
 
+## Release 2.14.4
+
+### New improved flow run graph with dependency layout
+
+The Flow Run Graph has been rebuilt from the ground up, offering significantly improved performance capabilities that allow larger flow runs to be displayed much more smoothly. We’ve added three new layouts: two non-temporal layout options, designed to provide a clearer picture of the dependency paths, and one to facilitate easy comparison of run durations. The x-axis can now be independently scaled for temporal layouts, you can adjust it in the graph settings or with the new keyboard shortcuts - and +. Additional small bug fixes, including displaying of cached tasks, have also been included.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:7]: I don't believe this should be capitalized
```suggestion
The flow run graph in the Prefect UI has been rebuilt from the ground up, offering significantly improved performance capabilities that allow larger flow runs to be displayed much more smoothly. We’ve added three new layouts: two non-temporal layout options, designed to provide a clearer picture of the dependency paths, and one to facilitate easy comparison of run durations. The x-axis can now be independently scaled for temporal layouts, you can adjust it in the graph settings or with the new keyboard shortcuts - and +. Additional small bug fixes, including displaying of cached tasks, have also been included.
```

Comment (serinamarie) [inline RELEASE-NOTES.md]:
```suggestion
The Flow Run Graph has been rebuilt from the ground up, offering significantly improved performance capabilities that allow larger flow runs to be displayed much more smoothly. We’ve added three new layouts: two non-temporal layout options, designed to provide a clearer picture of the dependency paths, and one to facilitate easy comparison of run durations. The x-axis can now be independently scaled for temporal layouts; and you can adjust it in the graph settings or with the new keyboard shortcuts - and +. Additional small bug fixes, including displaying of cached tasks, have also been included.
```

Comment (serinamarie) [inline RELEASE-NOTES.md]:
```suggestion
The Flow Run Graph has been rebuilt from the ground up, offering significantly improved performance capabilities that allow larger flow runs to be displayed much more smoothly. We’ve added three new layouts: two non-temporal layout options, designed to provide a clearer picture of the dependency paths, and one to facilitate easy comparison of run durations. The x-axis can now be independently scaled for temporal layouts, you can adjust it in the graph settings or with the new keyboard shortcuts - and +. We included additional small bug fixes, including the display of cached tasks.
```

File: RELEASE-NOTES.md [hunk 14]
```diff
@@ -1,5 +1,43 @@
...
+<p align="center">
+<img width="976" alt="flow run graph sequential grid view" src="https://user-images.githubusercontent.com/6776415/281769376-bccc4cd5-db2c-42b9-9c21-fc32b094323b.png">
+</p>
+
+See the following pull requests for implementation details:
+- Improved flow run graph with new dependency layout — https://github.com/PrefectHQ/prefect/pull/11112
+- Flow run graph v2 polish — https://github.com/PrefectHQ/prefect/pull/11105
+- Handle Cached tasks on graph-v2 — https://github.com/PrefectHQ/prefect/pull/11113
+- Exclude any flow/task run with a `NULL` `start_time` from Graph v2 — https://github.com/PrefectHQ/prefect/pull/11132
+- Use an unstarted flow run's expected start time for the Graph start time — https://github.com/PrefectHQ/prefect/pull/11138
```
Comment (urimandujano) [inline RELEASE-NOTES.md:18]:
```suggestion
- Use an unstarted flow run's expected start time as the Graph's start time — https://github.com/PrefectHQ/prefect/pull/11138
```

File: RELEASE-NOTES.md [hunk 15]
```diff
@@ -1,5 +1,43 @@
...
+
+<p align="center">
+<img width="976" alt="flow run graph sequential grid view" src="https://user-images.githubusercontent.com/6776415/281769376-bccc4cd5-db2c-42b9-9c21-fc32b094323b.png">
+</p>
+
+See the following pull requests for implementation details:
+- Improved flow run graph with new dependency layout — https://github.com/PrefectHQ/prefect/pull/11112
+- Flow run graph v2 polish — https://github.com/PrefectHQ/prefect/pull/11105
+- Handle Cached tasks on graph-v2 — https://github.com/PrefectHQ/prefect/pull/11113
+- Exclude any flow/task run with a `NULL` `start_time` from Graph v2 — https://github.com/PrefectHQ/prefect/pull/11132
```
Comment (serinamarie) [inline RELEASE-NOTES.md:17]: do users know about Graph v2? Can we just say "flow run graph" perhaps or something similar

File: RELEASE-NOTES.md [hunk 16]
```diff
@@ -1,5 +1,43 @@
...
+- Use an unstarted flow run's expected start time as the Graph's start time — https://github.com/PrefectHQ/prefect/pull/11138
```
Comment (serinamarie) [inline RELEASE-NOTES.md:18]:
```suggestion
- Use an unstarted flow run's expected start time as the flow run graph's start time — https://github.com/PrefectHQ/prefect/pull/11138
```

Comment (serinamarie) [inline RELEASE-NOTES.md:18]:
```suggestion
See the following pull requests for implementation details:
- https://github.com/PrefectHQ/prefect/pull/11112
- https://github.com/PrefectHQ/prefect/pull/11105
- https://github.com/PrefectHQ/prefect/pull/11113
- https://github.com/PrefectHQ/prefect/pull/11132
- https://github.com/PrefectHQ/prefect/pull/11138
```

File: RELEASE-NOTES.md [hunk 17]
```diff
@@ -1,5 +1,43 @@
...
+
+### Enhancements
+- Add API route for block counts — https://github.com/PrefectHQ/prefect/pull/11090
+- Improved tag handling on `DeploymentImage` for `.deploy`
```
Comment (serinamarie) [inline RELEASE-NOTES.md:22]:
```suggestion
- Improved tag handling on `DeploymentImage` for `.deploy`:
```

File: RELEASE-NOTES.md [hunk 18]
```diff
@@ -1,5 +1,43 @@
 # Prefect Release Notes
 
+## Release 2.14.4
+
+### New improved flow run graph with dependency layout
+
+The flow run graph in the Prefect UI has been rebuilt from the ground up, offering significantly improved performance capabilities that allow larger flow runs to be displayed much more smoothly. We’ve added three new layouts: two non-temporal layout options, designed to provide a clearer picture of the dependency paths, and one to facilitate easy comparison of run durations. The x-axis can now be independently scaled for temporal layouts, you can adjust it in the graph settings or with the new keyboard shortcuts - and +. Additional small bug fixes, including displaying of cached tasks, have also been included.
```
Comment (serinamarie) [inline RELEASE-NOTES.md]:
```suggestion
The flow run graph in the Prefect UI has been rebuilt from the ground up, offering significantly improved performance capabilities that allow larger flow runs to be displayed much more smoothly. We’ve added three new layouts: two non-temporal layout options, designed to provide a clearer picture of the dependency paths, and one to facilitate easy comparison of run durations. The x-axis can now be independently scaled for temporal layouts; and you can adjust it in the graph settings or with the new keyboard shortcuts - and +. We've also included small bug fixes, including the display of cached tasks.
```
^ sounds more active

**Review by serinamarie — APPROVED** [review]
LGTM!

**Pull request merged**